scripts/ma_dictionaries.py

```diff
@@ -333,6 +333,43 @@
         "wash": "Water Sanitation and Hygiene"
     }
 
+FeatureCategoryCodes = {
```

Comment:
if this values same as in https://github.com/konturio/geocint-mapaction/blob/main/static_data/directories/directories.csv
maybe we could use them? To prevent possible errors in the future

Reply:
Yes, it's the same information that are in directories.csv. However, I do not see how I can use directories.csv directly from python code (without parsing, which is not a good idea two). Maybe we could generate directories.csv from python. I would prefer a separate task for it.